task3/index.html

```diff
@@ -0,0 +1,254 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Gektor</title>
+    <link rel="stylesheet" href="styles.css">
+
+</head>
+        <body>
+            <div class="leftcol">
+                <header>
+                    <img class="logo" src="logo.jpg" align="center" alt="logotip"> <strong>Gektor</strong>
+                    <p class="username"><strong>Anastasiya Zyatkova</strong></p>
+                </header>
+                <div id="menu">
```

Comment:
First of all it's a navigation - so we have a nav element for that purpose.
Second thing is that we have a list of navigation items, so in ideal world we need to reflect in html the fact that it is list.
And the third thing each of menu item is leads to some page, it should be some actionable thing like link
So ideal structure will look like
```
<nav class="menu">
  <ul>
    <li class="menu-item">
      <a class="menu-link" href="#home">Home</a>
     </li>
     ...
</ul>
</nav>
```
So we make a little more markup, but its provides semantics, which helps users and will helps us generally.